Client/Frontend/Home/RecentTabsPanel.swift

```diff
@@ -0,0 +1,162 @@
+        let headerLabel = UILabel()
+        headerLabel.text = Strings.RecentTabsPanelTitle
+        headerLabel.font = DynamicFontHelper.defaultHelper.DeviceFontHistoryPanel
+        headerLabel.textAlignment = .Center
+        headerLabel.backgroundColor = UIColor.whiteColor()
+        return headerLabel
+    }()
+    private lazy var tableViewController: RecentTabsPanelSiteTableViewController = {
+        return RecentTabsPanelSiteTableViewController()
+    }()
+    private lazy var historyBackButton: HistoryBackButton = {
+        let button = HistoryBackButton()
+        button.addTarget(self, action: #selector(RecentTabsPanel.historyBackButtonWasTapped), forControlEvents: .TouchUpInside)
+        return button
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        view.backgroundColor = UIColor.whiteColor()
+
+        tableViewController.profile = self.profile
+        tableViewController.homePanelDelegate = homePanelDelegate
+        tableViewController.historyPanel = historyPanel
+
+        self.addChildViewController(tableViewController)
+        self.view.addSubview(tableViewController.view)
+        self.view.addSubview(historyBackButton)
+        self.view.addSubview(recentlyClosedHeader)
+
+        historyBackButton.snp_makeConstraints { make in
+            make.top.left.right.equalTo(self.view)
+            make.height.equalTo(50)
+            make.bottom.equalTo(recentlyClosedHeader.snp_top)
+        }
+
+        recentlyClosedHeader.snp_makeConstraints { make in
+            make.top.equalTo(historyBackButton.snp_bottom)
+            make.height.equalTo(20)
+            make.bottom.equalTo(tableViewController.view.snp_top).offset(-10)
+            make.left.right.equalTo(self.view)
+        }
+
+        tableViewController.view.snp_makeConstraints { make in
+            make.left.right.bottom.equalTo(self.view)
+        }
+
+        tableViewController.didMoveToParentViewController(self)
+    }
+
+    @objc private func historyBackButtonWasTapped(gestureRecognizer: UITapGestureRecognizer) {
+        self.navigationController?.popViewControllerAnimated(true)
+    }
+}
+
+class RecentTabsPanelSiteTableViewController: SiteTableViewController {
+    weak var homePanelDelegate: HomePanelDelegate?
+    weak var historyPanel: HistoryPanel?
+    var recentlyClosedTabs: [String:[String]]!
+
+    init() {
+        super.init(nibName: nil, bundle: nil)
+        NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(HistoryPanelSiteTableViewController.notificationReceived(_:)), name: NotificationPrivateDataClearedHistory, object: nil)
+        NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(HistoryPanelSiteTableViewController.notificationReceived(_:)), name: NotificationDynamicFontChanged, object: nil)
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.accessibilityIdentifier = "Recently Closed Tabs List"
```

Comment:
Nit: No point in adding these identifiers until we're actually using them.

And on that note, this could use some UI tests!